hotdoc/extensions/gi/utils.py

```diff
@@ -200,16 +202,17 @@ def get_language_classes():
     Banana banana
     """
     all_classes = {}
-    deps_map = {}
 
-    for entry_point in pkg_resources.iter_entry_points(
-            group='hotdoc.extensions.gi.languages', name='get_language_classes'):
+    for entry_point in entry_points(
+            group='hotdoc.extensions.gi.languages',
+            name='get_language_classes'):
         try:
+            ep_name = entry_point.name
```

Comment:
This is here (despite `ep_name` never being used) because the results of `importlib.metadata.entry_points()` are [documented](https://importlib-metadata.readthedocs.io/en/latest/migration.html#pkg-resources-iter-entry-points) as not being validated; invoking any property is the suggested means of triggering validation.